src/components/App.tsx

```diff
@@ -10,11 +10,11 @@ interface Category {
 interface Endeavor {
 	id:number;
 	label:string;
-	category:number; // id from Category
+	category_id:number;
 }
 
 interface ActivityEndeavor {
-	id:number; // id from Endeavor
+	endeavor_id:number;
```

Comment:
i got confused about these opening it back up, thought the `_id` helped make them more explicit